doc/network_scenarios.md

```diff
@@ -0,0 +1,418 @@
+Introduction
+============
+
+This document tries to clarify some doubts and expectations related to AutoYast network
+configuration, specially will try to show how affect the `keep_install_network` option.
+
+We put an special attemption to that flag because there was a bug in SP1 which always 
+saved the network configuration even with `keep_install_network=false`. When we fixed 
+the bug, users who were using a SP1 profile started reporting bugs for SP2.
+
+The current documentation explains that the network configuration is done during the second
+stage (imported from the profile) as we can see in the image below.
+
+<p align="center">
+  <img src="https://www.suse.com/documentation/sles-12/singlehtml/book_autoyast/images/autoyast-oview.png" />
+</p>
+
+Nevertheless, during the first stage we could need network so we are able to configure
```

Comment:
*the network would be required so it's possible to configure it...*

And maybe the sentence is too long (what about splitting the decision from the rest?) Something like: "The configuration can be copied to the installed system if `keep_install_network` is set to `true`." (just a suggestion).